endpoints/openrtb2/sample-requests/valid-whole/exemplary/simple.json

```diff
@@ -26,7 +30,7 @@
           ]
         },
         "ext": {
-          "appnexus": {
+          "Appnexus": {
             "placementId": 12883451
```

Comment:
instead of updating [sample-requests/valid-whole/exemplary/simple.json](https://github.com/prebid/prebid-server/pull/3113/commits/3b753adb20068302acd49728fd2577b7b06da700#diff-f5804f6547484174adb4a4ffb407f07867bbf39504462e1d2030bbecf46ad285). How about adding new json test file `case-insensitive-bidder.json`

Reply:
https://github.com/prebid/prebid-server/pull/3113/commits/6e3ba92f8e2c7b93e7aab77969183a268529bc42